client/landing/stepper/utils/persist-query-params.ts

```diff
@@ -0,0 +1,8 @@
+export const persistQueryParams = ( url: string, urlQueryParams: URLSearchParams ) => {
```

Comment:
As you suggest, this should probably be called `appendQueryParams`. Because persist implies some type of storage like localStorage or such.

Reply:
Agree.